app/assets/javascripts/misc/i18n-strings.js.erb

```diff
@@ -46,11 +46,15 @@ window.LoginGov = window.LoginGov || {};
 ] %>
 
 window.LoginGov.I18n = {
+  currentLocale: function() { return this.__currentLocale || (this.__currentLocale = document.querySelector('html').lang); },
```

Comment:
Yes, we reflect the current locale back in as the `lang` attribute on the HTML element:

https://github.com/18F/identity-idp/blob/master/app/views/layouts/application.html.slim#L2

So whatever locale we render the page as will get set there and we'll pick it up here